addons/event/models/event_event.py

```diff
@@ -480,6 +482,27 @@ def _check_closing_date(self):
             if event.date_end < event.date_begin:
                 raise ValidationError(_('The closing date cannot be earlier than the beginning date.'))
 
+    def _set_default_event_mail_ids(self):
```

Comment:
Shouldn't we update templates (data and demo) if we think this should be standard ? Indeed if we propose to use templates and if we think mail are used in all events, probably by default event templates should use them.